content/terminal/openbb-copilot.md

```diff
@@ -5,215 +5,340 @@ description: Learn how to use OpenBB Copilot to interact with the OpenBB Termina
+- Iteratively manipulate and transform data and charts
+- Be used to search the web for additional information
+- Provide citations
 
 ## Introduction
 
+To use OpenBB Copilot, click on the purple icon located at the center right of the
+Terminal screen:
+
 <div style={{display: 'flex', justifyContent: 'center'}}>
 <img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/open_copilot.jpg" alt="open openbb copilot" width="70%" height="70%" />
 </div>
 
-To open OpenBB Copilot, click on the purple icon located at the center right of the
-Terminal screen.
+Under most circumstances, you can treat OpenBB Copilot like a personal research
+assistant. OpenBB Copilot has access to the widgets (including your uploaded
+files) on your currently-active dashboard, the OpenBB API and any custom backend
+endpoints you have added to OpenBB Terminal. 
 
-Under most circumstances, you can treat OpenBB Copilot like your very own
-personal research assistant. OpenBB Copilot has access to the widgets on your
-dashboard, the OpenBB API and any custom endpoints you have added, as well as
-your uploaded files. Using all of these sources, OpenBB Copilot can assist you in
-performing a wide variety of tasks.
+By utilizing all of these sources of data, OpenBB Copilot can assist you in
+performing a wide variety of research and analysis tasks.
 
 <div style={{display: 'flex', justifyContent: 'center'}}>
 <img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/copilot_opened.jpg" alt="opened openbb copilot" width="70%" height="70%" />
 </div>
 
-We'll be exploring each of these features in the following sections below.
 
-## How chats work
+## Understanding Chats
+
+OpenBB Copilot is a chat-based assistant that leverages multiple LLMs (Large
+Language Models) under-the-hood to understand your queries and provide answers. 
 
-OpenBB Copilot is a chat-based assistant that uses the conversation history to
-help answer your queries.  This allows you to ask follow-up questions or guide
-OpenBB Copilot while performing tasks.
+Since OpenBB Copilot is a chat-based assistant, it maintains a conversation history
+to provide contextual responses. This means you can have natural back-and-forth
+discussions, ask follow-up questions, and refine your queries based on previous
+responses. The conversation history helps OpenBB Copilot understand the context of
+your questions and provide more relevant and personalized assistance throughout
+your interaction.
 
 For example, if you ask about the price-to-earnings (P/E) ratio, and then follow
 up by asking about other important financial ratios, OpenBB Copilot will
-remember the context and suggest relevant ratios:
+understand the context and suggest relevant ratios:
 
 <div style={{display: 'flex', justifyContent: 'center'}}>
 <img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/pe_ratio_chat_example.jpg" alt="chat history openbb copilot" width="40%" height="40%" />
```

Comment:
Slightly outdated screenshot

![image](https://github.com/user-attachments/assets/27184417-86c5-4c86-9ce3-dc85292d196a)

Reply:
It also shows localhost -- which isn't good! I'll update. Nice catch.

Reply:
Fixed!